### PR TITLE
Update to Cancel Transfer dialog

### PR DESCRIPTION
**Pull request by shanimal08**
### References Issues:
#1555
### Description:
Checking for ticket count, updated wording
### Environment Variables
 * No change

### CODE REVIEW

**Comment by Jasonvdb** [comment]
Testing this from "My Events" but it's not displaying the ticket count in the text. I see you're handling that but in what scenario would ticket count not be set?
Screenshot 2019-07-24 at 10 08 40

**Comment by shanimal08** [comment]
I only updated the Fans stuff to send the prop for the time being, because I didn't want to mess with something like Refunds that might be changing soon 😬 so just added that in to be careful. But if it's fine/safe for me to go ahead and update all the places using the dialog now then I'll do that instead and take out the check?

**Comment by Jasonvdb** [comment]
Oh ok cool I see, yeah I think leave the check in to be safe. But if it's simple add the ticket count from where it's used in My events where the end users will see it

File: src/components/pages/myevents/EventTicketsCard.js [hunk 1]
```diff
@@ -267,7 +267,7 @@ class EventTicketsCard extends Component {
 								<Typography className={classes.ticketDetailsText}>
 									<StyledLink
 										underlined
-										onClick={() => onCancelTransfer(transfer_key)}
+										onClick={() => onCancelTransfer(transfer_key, index + 1)}
```
Comment (Jasonvdb) [inline src/components/pages/myevents/EventTicketsCard.js]: If I only cancel only the 5th ticket in the row it'll just be like "cancel transferring the 5 tickets"

**Pull request merged**